lib/nerves_hub_link.ex

```diff
@@ -55,4 +57,20 @@ defmodule NervesHubLink do
   """
   @spec send_file(Path.t()) :: :ok | {:error, :too_large | File.posix()}
   defdelegate send_file(file_path), to: Socket
+
+  @doc """
+  Send a health check to NervesHub
+  """
+  @spec send_health_check() :: Message.DeviceStatus.t()
+  def send_health_check() do
```

Comment:
the only thing that might be handy is catching errors here (since people can create their own reports) and send up an error message to hub, something about how there was an error generating the report?

Reply:
Interesting, yeah, somewhere in the chain, that should be a thing.

Reply:
That could work, although a little funny/odd that the service that generates report that uses alarms also generates alarms